bool usePose
geometry_msgs/Pose pose
geometry_msgs/Twist vel
---
bool success